crates/wie_wipi_java/src/base.rs

```diff
@@ -90,6 +90,7 @@ pub fn get_class_proto(name: &str) -> Option<JavaClassProto> {
     Some(match name {
         "java/lang/Exception" => crate::r#impl::java::lang::Exception::as_proto(),
         "java/lang/InterruptedException" => crate::r#impl::java::lang::InterruptedException::as_proto(),
+        "java/lang/IllegalArgumentException" => crate::r#impl::java::lang::IllegalArgumentException::as_proto(),
```

Comment:
```suggestion
        "java/lang/IllegalArgumentException" => crate::r#impl::java::lang::IllegalArgumentException::as_proto(),
        "java/lang/InterruptedException" => crate::r#impl::java::lang::InterruptedException::as_proto(),
```
Please sort this alphabetically